src/targets/gpu/compile_hip.cpp

```diff
@@ -176,14 +174,15 @@ compile_hip_src(const std::vector<src_file>& srcs, std::string params, const std
 {
     hiprtc_program prog(srcs);
     auto options = split_string(params, ' ');
+    options.push_back("-DMIGRAPHX_HIPRTC");
```

Comment:
Also `-DMIGRAPHX_HAS_DPP=0` needs to be added.

Reply:
Done.